tm5-700:
  # DH table for the TM5-700 6-Dof cobot 
  # D-H table (alpha_{i-1}, a_{i-1}, d_{i}, theta_{i})
  # Units: [deg, mm, mm, deg]
  dh_j1: [0.0, 0.0, 145.2, 0.0]
  dh_j2: [-90.0, 0.0, 0.0, -90.0]
  dh_j3: [0.0, 329.0, 0.0, 0.0]
  dh_j4: [0.0, 311.5, -122.3, -90.0]
  dh_j5: [-90.0, 0.0, 106.0, 0.0]
  dh_j6: [-90.0, 0.0, 113.15, 180.0]

  